mix.exs

```diff
@@ -46,6 +46,7 @@ defmodule Slax.Mixfile do
       {:mox, "~> 1.1", only: :test},
       {:jason, "~> 1.1"},
       {:ex_machina, "~> 2.2", only: :test},
+      {:sobelow, "~> 0.13"},
```

Comment:
A not-super-important optimization here is to only install sobelow in test and dev environments, and not have it available at runtime. The Sobelow docs suggest this:
```
{:sobelow, "~> 0.13", only: [:dev, :test], runtime: false}
```